### DISCUSSION TITLE
How do I get files in hidden folders to show up?

### DISCUSSION THREAD

**Discussion by IgnisDa** [helix-editor/helix#5458] [Q&A] [answered]
I have a few files in folders that start with `.`. For example: `.github/workflows/ci.yml`. Files like this do not show up in the fuzzy file finder (`:<space>f`). How can I get them to show up?

**Comment by the-mikedavis** [comment, marked as answer]
You can set the `hidden` option to `false` in the file-picker configuration to show hidden files: https://docs.helix-editor.com/master/configuration.html#editorfile-picker-section

If you're running from master you can also use `:open .github/workflows` to open a file picker for that directory despite it being hidden.
